### PR TITLE
Geoblocker

### PR DESCRIPTION
**Pull request by nichosystem**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [x] Code follows the style guide
- [x] I have performed a self-review of my own code
- [x] New and existing tests pass locally
- [x] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-h0d9p2wcr-templedao.vercel.app
https://pr-833-preview.templedao.link

Built with commit 4010b4b6f273e951368c32762ef25080fdda114e.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

File: apps/dapp/src/components/Layouts/CoreLayout/Account.tsx [hunk 1]
```diff
@@ -23,13 +23,28 @@ export const Account = () => {
...
+    };
+    checkBlocked();
+  }, []);
 
   if (accountLoading || connectLoading || networkLoading) {
     return <Loader />;
   }
 
+  if (isBlocked) {
+    return <ConnectButton disabled={true} label="Restricted Jurisdiction" isSmall isActive isUppercase role="button" />;
```
Comment (marshall2112) [inline apps/dapp/src/components/Layouts/CoreLayout/Account.tsx:45]: Nit: prefer to omit explicit `true`, i.e. `<ConnectButton disabled label=....`

**Review by marshall2112 — CHANGES_REQUESTED** [review]
I have two small pieces of feedback (nits, not blocking) and a question.

File: apps/dapp/api/geoblock.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const handler = (request, response) => {
+  response.status(200).json({
+    country: request.headers['x-vercel-ip-country'],
+    blocked: ['US', 'CN', 'RU', 'KP', 'IR'].includes(request.headers['x-vercel-ip-country']),
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/api/geoblock.js:4]: let's skip for now and revisit if it becomes a painpoint

**Review by marshall2112 — APPROVED** [review]
LGTM

**Pull request merged**